testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -60,16 +61,26 @@ class TopologyAttrMixin(object):
     Ridx = np.array([0, 0, 2, 2, 1, 1, 3, 3, 1, 2])
     Sidx = np.array([0, 1, 1, 0])
 
+    # single_idx = np.array([0])
+
     def setUp(self):
         self.top = Topology(10, 4, 2,
                             attrs=[self.attrclass(self.values.copy())],
                             atom_resindex=self.Ridx,
                             residue_segindex=self.Sidx)
-
         self.attr = getattr(self.top, self.attrclass.attrname)
 
+        # TODO: should we treat single element attr specially. We do that
```

Comment:
I think if you make a `DummyGroup` with an integer not a list/array of values then test using that, so

```
DummySeg = DummyGroup(1)

self.attr.get_segments(DummySeg)

```
Should give you the sum of the Segment.

So like this test but replace `[1]` with `1`

https://github.com/kain88-de/mdanalysis/blob/6b33051768d67e3603be4d3cfc6a81e54c9a46af/testsuite/MDAnalysisTests/core/test_topologyattrs.py#L152